packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Docker/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Collapsible,
+  Text,
+  Button,
+  Stack,
+  Link,
+} from '@codesandbox/components';
+import { useUpgradeFromV1ToV2 } from 'app/hooks/useUpgradeFromV1ToV2';
+
+export const Docker = () => {
+  const { perform, loading, canConvert } = useUpgradeFromV1ToV2('Docker menu');
+
+  return (
+    <Collapsible title="Docker" defaultOpen>
+      <Stack direction="vertical" gap={6} padding={2}>
+        <Text as="p" variant="muted" margin={0}>
+          To run Docker, you need to upgrade your Browser Sandbox into a Cloud
+          Sandbox.
+        </Text>
+        <Text as="p" variant="muted" margin={0}>
+          Cloud Sandboxes are an improved coding experience that run your code
+          in the cloud. They allow you to run Docker, code in new languages, add
+          servers, databases, and much more.
+        </Text>
+        <Text as="p" variant="muted" margin={0}>
+          Do you want to convert this sandbox?
+        </Text>
+        <Stack align="center" gap={4}>
+          <Button
+            onClick={perform}
+            loading={loading}
+            variant="primary"
+            autoWidth
+          >
+            Yes, {canConvert ? 'convert' : 'fork and convert'}
+          </Button>
```

Comment:
I noticed that there's no error handling if converting fails. Took a look and seems that the hook does not catch errors so I created a task in Linear: https://linear.app/codesandbox/issue/XTD-698/handle-errors-when-upgrading-from-v1-to-v2. The priority is medium but I believe we can increase it next week.

Reply:
Really good one!